src/cls/queue/cls_queue_const.h

```diff
@@ -3,7 +3,7 @@
 
 #define QUEUE_CLASS "queue"
```

Comment:
if these #defines are getting annoying (long), then maybe put in namespace and use static constexpr std::string?

Reply:
I tried using constexpr string_view for these #defines, but cls_register_cxx_method expects a const char*, the constexpr can be converted to const char * using std::string(string_view_object).c_str(), but is this really required?

Reply:
nah, probably not